src/realm-settings/EmailTab.tsx

```diff
@@ -93,6 +97,9 @@ export const RealmSettingsEmailTab = ({
       serverSettings[key] = mapperFn(serverSettings[key]);
     }
 
+    // For default value, back end is expecting null instead of 0
+    if (serverSettings["port"] === 0) serverSettings["port"] = null;
```

Comment:
Should be able to do a regular property access here.
```suggestion
    if (serverSettings.port. === 0) serverSettings.port = null;
```

Reply:
I tried that.  VS Code didn't recognize the property.

serverSettings is of type `[x: string]: any;`

Reply:
Seemed to work here, I added a commit for it.